File: Modul-01/c/PRAK108-2310817210009-MuhammadIbnuSina.c
#include <stdio.h>

int main() {
    // Deklarasi variabel
    float Kilometer, putaran, jariJari;

    // Input jarak tempuh dan putaran
        printf("Diketahui :\nPak Dengklek mengelilingi taman = ");
    scanf("%f Putaran", &putaran);

    printf("Jarak Tempuh Pak Dengklek = ");
    scanf("%f Kilometer", &Kilometer);

    // Menghitung jari-jari taman
    jariJari = Kilometer / (2 * 3.14159265359 * putaran);

    // Menampilkan hasil
    printf("\nJawaban :\nJari-jari taman yang dikelilingi Pak Dengklek adalah %.2f Kilometer\n", jariJari);

    return 0;
}

File: Modul-01/python/PRAK108-2310817210009-MuhammadIbnuSina.py
# Meminta input jumlah putaran dan jarak tempuh dari pengguna
putaran_str = input("Diketahui :\nPak Dengklek mengelilingi taman = ")
jarak_tempuh_str = input("Jarak tempuh Pak Dengklek = ")

putaran_str = ''.join(filter(str.isdigit, putaran_str))
jarak_tempuh_str = ''.join(filter(str.isdigit, jarak_tempuh_str))

# Mengonversi input menjadi float
putaran = float(putaran_str)
jarak_tempuh = float(jarak_tempuh_str)

# Menampilkan hasil dengan teks "Putaran" dan "Kilometer"
print(
    f"\nJawaban :\nJari-jari taman yang dikelilingi Pak Dengklek adalah {jarak_tempuh / (2 * 3.14 * putaran):.2f} Kilometer")
